hkml_list.py

```diff
@@ -703,7 +703,8 @@ def set_argparser(parser=None):
     parser.add_argument(
             '--source_type',
             choices=['mailing_list', 'mbox', 'clipboard', 'tag'],
-            help='type of sources')
+            help='type of sources',
+            default='mailing_list')
```

Comment:
Thank you for this great pull request.

The intention is to support multiple source types without asking for additional options for users.  `hkml list` will then infer the type on its own.  With this change, we may ask users to provide this option always for none-mailing list cases.

If the problem is the unkind error message, what about making the error message when inferring the type on `get_mails()` failed?

Reply:
Makes sense. What do you think about this new version?